DOCS-10084 landing page (#128)
mulesoft/docs-exchange: modules/ROOT/pages/index.adoc

:imagesdir: ../assets/images/
:page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc

> Anypoint Exchange is your curated catalog of APIs and other assets, including API groups, API specification fragments, policies, and more.

Reviewer: Can we elaborate a bit more on the value and specify assets beyond just APIs as well. Something on the lines of "Anypoint Exchange is a catalog of reusable assets such as APIs, Policies, API spec fragments, API groups as well as other integration assets such as connectors, templates, examples and custom assets. Anypoint Exchange can be used to catalog (publish), share, discover, learn about, and reuse assets within your organization, to drive collaboration, boost productivity and promote standards."

Reviewer 2: done

> Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization that drive collaboration, boost productivity, and promote standards.

Reviewer 3: ```suggestion
Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API specification fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (or _publish_), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.
```

image::ex2-ui.png[Exchange interface]

> Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.

Reviewer 3: ```suggestion
Use Exchange to discover, share, and incorporate assets into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.
```

Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.

After you log into Exchange, you can create assets for APIs, connectors, templates, examples, API Notebooks, and custom assets.

== Try APIs

. Click All Types > REST APIs and choose any API.
. Click the asset and notice how the API functions appear in the left
navigation area. Exchange lets you view the functions in an API.
. Click a function and more information appears. At the right side of the screen, you can test the API by entering sample values to send to the API and get responses from the API.

== Need More Information on an Asset?

Click the Helpful Links item in the left navigation area for links to documentation.

== Exchange Content

Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API Groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.

> MuleSoft certifies its public content and reviews and approves partner connectors. All private content is internal to an organization and not visible outside that organization. Customers are responsible for any private content they publish within their Exchange instance.

Reviewer: I would want to retain some version of this. something on the lines of:

Assets under the 'Provided by Mulesoft' organization are created by Mulesoft and its partners, and are certified by Mulesoft. These assets are publicly available to all customers as well. However, all assets that are created by an organization and published into the organization's instance of Exchange, are private and not visible outside that organization.... Customers are responsible for any private content they publish within their organization's Exchange instance.

Reviewer 2: Done. added this to line 18 in the index.adoc.

== Language Support
Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.

Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.

> [[content-types]]

Reviewer 2: hi, is the removed content no longer applicable or did you just streamline the landing page since the content is located in another topics?

Author: This PR moves the "Content Types" list from the landing page to the new page "Asset Types". It never really made sense for it to be part of the landing page.

Reviewer: Callout - If the asset types section was linked anywhere else in the product/documentation it will need to be updated to point to the new section instead.

> Assets under the *Provided by MuleSoft* organization are created by MuleSoft and its partners, and are certified by Mulesoft. These assets are publicly available to all customers as well. However, all assets that are created by an organization and published into the organization's instance of Exchange, are private and not visible outside that organizationCustomers are responsible for any private assets they publish within their Exchange instance.

Reviewer 3: ```suggestion
Assets displayed as *Provided by MuleSoft* are created by MuleSoft and its partners and certified by MuleSoft. These assets are publicly available to all customers. However, any assets created by an organization and published into that organization's instance of Exchange are private: not visible to customers outside of that organization. Organization customers are responsible for any private assets they publish within their Exchange instance.
```

> This page applies to the current version, Exchange 2. Documentation on Exchange 1 is at xref:about-exchange1.adoc[About Exchange 1 (Old Version)]. We encourage Exchange 1 users to xref:migrate.adoc[migrate] to Exchange 2. The full version history is available in the xref:release-notes::exchange/anypoint-exchange-release-notes.adoc[Anypoint Exchange Release Notes].

Reviewer 3: We never overtly refer to units of information as pages, topics, subtopics, or any similar terms.
I suggest incorporating this information at the very beginning of the text introducing the page, something like "Before <blah blah blah>, you must be running Exchange <v.2.0?> or later.
OR
 Incorporate it as a Note at the end of that introduction:
```suggestion
NOTE: This information applies to the Exchange [v2.0 and later]current version. To get more information about Anypoint Exchange versioning, see xref:release-notes::exchange/anypoint-exchange-release-notes.adoc[Anypoint Exchange Release Notes].
```
